src/box/lua/integrity.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2023, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#pragma once
+
+#include "trivia/config.h"
+
+#if defined(ENABLE_INTEGRITY)
```

Comment:
Minor: Why not just use `#ifdef` here?

Reply:
I wanted to use the same style as for the other EE modules, for more details of this style see https://github.com/tarantool/tarantool/pull/8621.

Reply:
OK, thanks for clarification!